### ISSUE TITLE
Agenda yml files added 

### ISSUE THREAD

**Issue by U2NG** [uc-love-data-week/uc-love-data-week.github.io#12]
I updated and added yml files to add the schedule of events for each day.

**Comment by kekoziar** [comment]
it occurs to me that doing that for everything is pretty onerous and duplicative to the workshop.yml page.  What do you think about adding a `day` field in the `workshop.yml` posts, then rewrite the logic to build the schedule from that?  That way we only need to update one file.

**Comment by U2NG** [comment]
I was thinking the same thing while playing around with the agenda list.   A day field on the workshop list would be a good way to organize the order of the workshops and it would be great to only have to update 1 file.

**Comment by kekoziar** [comment]
I added the logic to build it from one file. well, actually two. 
In `workshops.yml`, I changed `date` to `day` and `time`, where day is the lower case single/double letter representation of the day of the week.
For example,
```
day: t
time: 1 - 2 pm
```

I also added a `dates.yml` file that has days of the event in the following structure:

```
- date: Feb 8, 2021
  day-full: Monday
  day-abbv: m 
```

The agenda builds based on a `for` loop over `dates.yml`, and adds workshop info when `workshops.day` is matched with 
`dates.day-abbv`.  The same logic and files are also used to update the workshop information.
